dbt-snowflake/src/dbt/include/snowflake/macros/relations/dynamic_table/describe.sql

```diff
@@ -30,6 +31,8 @@
         {% set _ = results.update({'catalog': run_query(_get_describe_iceberg_catalog_sql(relation))}) %}
     {% endif %}
 
+    ALTER SESSION UNSET QUOTED_IDENTIFIERS_IGNORE_CASE;
```

Comment:
is there any chance this causes issues with other models if we re-use connections? Does this impact globally setting  `QUOTED_IDENTIFIERS_IGNORE_CASE`?

Reply:
This only alters it for the session. And it appears a session is short lived for us. I needed to use a pre-hook in the test just for it to run as part of the same session. Simply running the statement before running dbt did not "stick". I think the risk is relatively low that it would impact anything else.